packages/preview/songting-book/0.0.3/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024

```

Comment:
I think you are required to have your name (or at least username) on this line. It would also be better to update the year to `2025` or `2024-2025`.